### PR TITLE
ERMS-4284: Frontend umstellen auf Semantic

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: server/gokbg3/grails-app/assets/javascripts/wekb.js [hunk 1]
```diff
@@ -52,18 +57,15 @@ $(function () {
...
 
     $('.xEditableValue').editable({
-
-        highlight: false,
-        language: "en",
         format:   "yyyy-MM-dd",
         validate: function(value) {
             if ($(this).attr('data-format') && value) {
-                if(! (value.match(/^\d{1,2}\.\d{1,2}\.\d{4}$/) || value.match(/^\d{4}-\d{1,2}-\d{1,2}$/)) ) {
+                if(! (value.match(/^\d{4}-\d{1,2}-\d{1,2}$/)) ) {
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/assets/javascripts/wekb.js:68]: why omitting German date format?

File: server/gokbg3/grails-app/services/com/k_int/ESSearchService.groovy [hunk 2]
```diff
@@ -116,7 +116,7 @@ class ESSearchService{
 
   def search(params, field_map){
     log.debug("ESSearchService::search - ${params}")
-
+println(params)
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/services/com/k_int/ESSearchService.groovy]: needed for control?

**Review by croqueGrec09 — COMMENTED** [review]
remarks

File: server/gokbg3/grails-app/assets/javascripts/wekb.js (hunk 1 above)
Comment (Moetez16) [reply server/gokbg3/grails-app/assets/javascripts/wekb.js:68]: Because all is in en. No prio now for translate or something else.

**Pull request merged**